perfkitbenchmarker/providers/gcp/gce_virtual_machine.py

```diff
@@ -128,9 +128,8 @@ def _GetOptionDecoderConstructions(cls):
 class MachineTypeDecoder(option_decoders.TypeVerifier):
   """Decodes the machine_type option of a GCE VM config."""
 
-  def __init__(self, option, **kwargs):
-    super(MachineTypeDecoder, self).__init__(option, (basestring, dict),
-                                             **kwargs)
+  def __init__(self, **kwargs):
```

Comment:
Do you need to update some of the error messages in MemoryDecoder now that self.option might be None?